app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -65,18 +65,26 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
     .object()
     .nullable()
     .shape({
-      label: yup.string().oneOf(ASSIGNED_CONTACT_OPTIONS.map(tag => tag.label)),
-      value: yup.string().oneOf(ASSIGNED_CONTACT_OPTIONS.map(tag => tag.value)),
+      label: yup
+        .string()
+        .oneOf(ASSIGNED_CONTACT_OPTIONS.map(contact => contact.label)),
+      value: yup
+        .string()
+        .oneOf(ASSIGNED_CONTACT_OPTIONS.map(contact => contact.value)),
     })
     .required("Assigned Contact is required"),
   tags: yup
     .array(
-      yup.object().shape({
-        label: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.label)),
-        value: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.value)),
-      })
+      yup
+        .object()
+        .nullable()
+        .shape({
+          label: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.label)),
+          value: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.value)),
+        })
     )
     .nullable()
+    .min(1, "Tag is required")
```

Comment:
Nullable is not required as the initial value is `[]` now. Please read upon what `nullable` does.